odl/solvers/functional/functional.py

```diff
@@ -884,3 +885,44 @@ def convex_conj(self):
         """
         return self.functional.convex_conj.translated(
             self.linear_term)
+
+
+class FunctionalDefaultConvexConjugate(Functional):
+
+    """The ``Functional`` representing ``f^*(.)``.
+
+    Does not provide a way in which to evaluate the functional, intended to
+    be used for its `proximal`.
+    """
+
+    def __init__(self, func):
+        """Initialize a new instance.
+
+        Parameters
+        ----------
+        func : `Functional`
+            Function corresponding to F.
+        """
+        if not isinstance(func, Functional):
+            raise TypeError('`func` {} is not a `Functional` instance'
+                            ''.format(func))
+
+        super().__init__(space=func.domain,
+                         linear=func.is_linear)
+
+        self.__functional = func
```

Comment:
Why double underscore here, but not [here](https://github.com/odlgroup/odl/pull/587/files#diff-50e0484cf75f430f06878917e0952f87R499)?